pomegranate/base.pyx

```diff
@@ -308,41 +271,31 @@ cdef class GraphModel( Model ):
 		self.d = 0
 
 	def __str__(self):
-		"""
```

Comment:
+1 for making these be a single line, but ultimately they should be like the ones above, with Parameters and Return sections. That may be on me to write, though, since this is just a formatting PR